hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -131,6 +142,13 @@ public void initialize(URI name, Configuration conf) throws IOException {
         // If port number is not specified, read it from config
         omPort = OmUtils.getOmRpcPort(conf);
       }
+    } else if (OmUtils.isServiceIdsDefined(conf)) {
+      // When host name or service id is given, and ozone.om.service.ids is
```

Comment:
We arrived at this point means, hostname or service id is not provided in url right?

Reply:
Correct. This is the case for command `ozone fs -ls o3fs://bucket.volume/` when HA is enabled.